.github/workflows/reviews.yml

```diff
@@ -0,0 +1,103 @@
+name: "Request reviews"
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_run:
+    workflows: [Pull Request Labeler]
+    types: [completed]
+jobs:
+  request-reviewer:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            const reviewers = []
+
+            // The number of the pull request that triggered this run. If label
+            // was added manually by a person the number will be stored in current
+            // context, otherwise the number will be stored in the payload.
+            const pr_number = context.issue.number || context.payload.workflow_run.pull_requests[0].number
+
+            const pr_data = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: pr_number
+            })
+            const labels = pr_data.data.labels.map(e => e.name)
+
+            if (labels.includes('api')) {
+              reviewers.push("bfredl")
```

Comment:
Any existing reviewers won't be touched/removed anyway, this is strictly an additive(?) operation. This will still work of course but isn't necessary. I'll let you make the call if you still prefer to change it.

Reply:
Ah, the API just requests new reviewers. Ok, then ignore my comments. :-)